src/test/java/com/google/ehub/utility/UtilsTest.java

```diff
@@ -7,28 +7,84 @@
 
 @RunWith(JUnit4.class)
 public final class UtilsTest {
+  private static final String DATE_FORMAT = "dd/MM/yyyy";
+  private static final String INVALID_DATE_FORMAT = "dddmmmmdczyyy";
+  private static final String VALID_DATE = "23/08/2013";
+  private static final String DATE_WITH_CONFLICTING_FORMAT = "23 Jul 2013";
+
+  private static final Long CORRECT_TIMESTAMP = 1377216000000L;
+
   @Test
-  public void isNextHigherEmptyStringValid() {
+  public void getNextHigherStringWithEmptyParam_ReturnsEmptyString() {
     Assert.assertEquals("", Utils.getNextHigherString(""));
   }
 
   @Test
-  public void isNextHigherSingleCharacterValid() {
+  public void getNextHigherStringWithSingleLetterParam_ReturnsNextLetter() {
     Assert.assertEquals("b", Utils.getNextHigherString("a"));
   }
 
   @Test
-  public void isNextHigherFullStringValid() {
+  public void getNextHigherStringWithCappedParam_ReturnsCappedParamWithNewAppendedLetter() {
     Assert.assertEquals("zzzzza", Utils.getNextHigherString("zzzzz"));
   }
 
   @Test
-  public void isNextHigherRegularStringValid() {
+  public void getNextHigherStringWithDefaultValidParam_ReturnsParamWithIncreasedLastLetter() {
     Assert.assertEquals("helloworle", Utils.getNextHigherString("helloworld"));
   }
 
   @Test
-  public void isNextHigherNullStringValid() {
+  public void getNextHigherStringWithNullParam_ReturnsEmptyString() {
     Assert.assertEquals("", Utils.getNextHigherString(null));
   }
+
+  @Test
+  public void getTimestampMillisFromDateWithNullDate_ReturnsNull() {
+    Assert.assertEquals(null, Utils.getTimestampMillisFromDate(null, DATE_FORMAT));
```

Comment:
```suggestion
    Assert.assertNull(Utils.getTimestampMillisFromDate(null, DATE_FORMAT));
```

Here and below

Reply:
OK I used assertNull() to shorten the method.